code/modules/mob/living/simple_animal/friendly/drone/drone_tools.dm

```diff
@@ -16,6 +16,7 @@
 		/obj/item/wrench/drone,
 		/obj/item/weldingtool/drone,
 		/obj/item/wirecutters/drone,
+		/obj/item/multitool/drone,
 	)
 	atom_storage.max_total_storage = 40
 	atom_storage.max_specific_storage = WEIGHT_CLASS_NORMAL
```

Comment:
The line below this, you have to increase their slots by one

Reply:
if you don't mind the comment, drones also cant use pipes system like air or disposals too so maybe a Rapid Pipe Dispenser too, and access to air alarms, for same reasons as the door, the drones can't build rooms with air alarms or systems because of this.

Reply:
They get access to air alarms if the CE grants it to them through BotKeeper or whatever it's called.